open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,13 @@
   },
```

Comment:
<p>Then you should call this function at line 5  as below</p><p> </p><p><code>filter(infected, 'Country/Region', 'China')</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Finished</p><p> </p>